application/extensions/admin/grid/CLSGridView.php

```diff
@@ -72,14 +80,31 @@ protected function lsAfterAjaxUpdate(): void
         }
     }
 
+    protected function rowLink()
```

Comment:
Missing phpdoc

Reply:
You might want to move up $options = [] to top and return $options at bottom, to always return an array instead of array|void.